README.md

```diff
@@ -1 +1,1205 @@
+[main]     Step 3/7 : COPY package.json package-lock.json ./
+[main]      ---> 6af34973ecb0
+[main]     Removing intermediate container d7b84160d116
+[main]     Step 4/7 : RUN npm install --ci --production     && npm cache clean --force     && rm -f /tmp/*
+[main]      ---> Running in e0a52c6e7d2e
+[main]     added 50 packages from 37 contributors and audited 126 packages in 2.53s
+[main]     found 0 vulnerabilities
+[main]     npm
+[main]      WARN using --force
+[main]      I sure hope you know what you are doing.
+[main]
+[main]      ---> 9ff0e005febd
+[main]     Removing intermediate container e0a52c6e7d2e
+[main]     Step 5/7 : COPY . ./
+[main]      ---> 211ef0e15576
+[main]     Removing intermediate container 75e2127552bf
+[main]     Step 6/7 : RUN cat /run/secrets/my-secrets
+[main]      ---> Running in 949098cba8b6
+[main]     This file has build-time secrets!
+[main]      ---> 6da25ec8f94f
+[main]     Removing intermediate container 949098cba8b6
+[main]     Step 7/7 : CMD npm start
+[main]      ---> Running in 1ae564a36e44
+[main]      ---> e590faac0013
+[main]     Removing intermediate container 1ae564a36e44
+[main]     Successfully built e590faac0013
+[Info]     Uploading images
+[Success]  Successfully uploaded images
+[Info]     Built on arm03
+[Success]  Release successfully created!
+[Info]     Release: cedeff39dc537c9cd4c1df3b97ffcbc7 (id: 1055448)
+[Info]     ┌─────────┬────────────┬────────────┐
+[Info]     │ Service │ Image Size │ Build Time │
+[Info]     ├─────────┼────────────┼────────────┤
+[Info]     │ main    │ 208.47 MB  │ 11 seconds │
+[Info]     └─────────┴────────────┴────────────┘
+[Info]     Build finished in 46 seconds
+			    \
+			     \
+			      \\
+			       \\
+			        >\/7
+			    _.-(6'  \
+			   (=___._/` \
+			        )  \ |
+			       /   / |
+			      /    > /
+			     j    < _\
+			 _.-' :      ``.
+			 \ r=._\        `.
+			<`\\_  \         .`-.
+			 \ r-7  `-. ._  ' .  `\
+			  \`,      `-.`7  7)   )
+			   \/         \|  \'  / `-._
+			              ||    .'
+			               \\  (
+			                >\  >
+			            ,.-' >.'
+			           <.'_.''
+			             <'
+```
+As you can see, step 6 output:
+```
+[main]     Step 6/7 : RUN cat /run/secrets/my-secrets
+[main]      ---> Running in 949098cba8b6
+[main]     This file has build-time secrets!
+```
+which is the contents of the secrets file. This file could obviously contain
+a raft of different functions, including a script that gets executed, text
+for filling in files, etc. As well as defining globally accessible secrets
+(which are shared to all services being built), there is also the option to
+define secrets that are only accessible to particular services, or to map them
+to different paths. This becomes useful in multicontainer build scenarios. This
+can be achieved by adding a `services` section to the `balena.yml` file. For
+example:
+```
+services:
+  main:
+    - source: main-only-secrets
+	  dest: my-main-secrets
```

Comment:
indentation incorrect here, `dest` should be at same level as `source`

Reply:
I think this a reader issue, in my editor they line up.